web-components/src/components/form/ControlledFormLabel.tsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import { Box } from '@mui/material';
-import { Body, Subtitle } from '../typography';
+import { Subtitle } from '../typography';
 
 interface Props {
   children: React.ReactNode;
   disabled?: boolean;
-  optional?: boolean;
-  labelSubText?: string;
+  optional?: boolean | string;
```

Comment:
`labelSubText` was only being used in the media forms, and should replace what's currently in `optional` but was being rendered underneath, which was different than the mockups. 

This allows the `optional` prop to accept a string which it can render instead of the default `(optional)` text